### ISSUE TITLE
Keyframes Defined as Export Vars Cause Null Exception

### ISSUE THREAD

**Issue by WoofMeowEtc** [styled-components/vue-styled-components#75]
Keyframes is working within the file where they are imported, but if you attempt to split out your animation concerns into a helper file, `vue-styled-components` will cause a null dereference.  Stack trace:

```
vue-styled-components.es.js?9c56:379 Uncaught TypeError: Cannot read property 'appendChild' of undefined
    at StyleSheet.insert (vue-styled-components.es.js?9c56:379)
    at StyleSheet$$1.insert (vue-styled-components.es.js?9c56:465)
    at keyframes (vue-styled-components.es.js?9c56:583)
    at eval (keyframes.js?c8f1:3)
    at Module../src/keyframes.js (app.js:3004)
    at __webpack_require__ (app.js:767)
    at fn (app.js:130)
```

Here is the area where it looks to be breaking:

```
// StyleSheet function
    key: 'insert',
    value: function insert(rule) {
      var insertedRule = void 0;

      if (isBrowser) {
        // this is the ultrafast version, works across browsers
        if (this.isSpeedy && this.sheet.insertRule) {
          this._insert(rule);
        } else {
          var textNode = document.createTextNode(rule);
          last(this.tags).appendChild(textNode);
          insertedRule = { textNode: textNode, appendRule: function appendRule(newCss) {
              return textNode.appendData(newCss);
            } };

          if (!this.isSpeedy) {
            // sighhh
            this.sheet = sheetForTag(last(this.tags));
          }
        }
      } else {
        // server side is pretty simple
        insertedRule = this.sheet.insertRule(rule);
      }

      this.ctr++;
      if (isBrowser && this.ctr % this.maxLength === 0) {
        this.tags.push(makeStyleTag());
        this.sheet = sheetForTag(last(this.tags));
      }
      return insertedRule;
```

The actual error is on this line:

```
          last(this.tags).appendChild(textNode);
```

Maybe this.tags is undefined?

**Comment by m-sebastiyan** [comment]
Hello!

Have you found a solution?

**Comment by liqueflies** [comment]
Can you share your helper code?

**Comment by m-sebastiyan** [comment]
I have a feeling that this was fixed in this PR: #79

**Comment by liqueflies** [comment]
Merged in **1.4.9**
<3
